play-routes-compiler-cli/play-routes-compiler-cli.bzl

```diff
@@ -0,0 +1,63 @@
+load("@rules_java//java:java_binary.bzl", "java_binary")
+load("@rules_java//java:java_import.bzl", "java_import")
+load("@rules_jvm_external//:defs.bzl", "java_export")
+load("@rules_scala_annex//rules:scala.bzl", "scala_library")
+load("//:workspace.bzl", "play_version")
+
+def generate_play_routes_compiler_targets(scala_version):
+    # For example 2.13 -> 2_13 or 2-13
+    scala_version_underscore = scala_version.replace(".", "_")
+    scala_version_dash = scala_version.replace(".", "-")
+    if scala_version.startswith("3"):
+        scala_library_target = "@play_routes_compiler_cli_{}//:org_scala_lang_scala3_library_3".format(scala_version_underscore)
```

Comment:
I had a comment for this and apparently deleted it.

Added it back
```        
        # Some of these deps are transitively included, e.g., the Scala library.
        # We're explicitly listing them here because we need them to show up in
        # the deps list in maven.
```

Reply:
Ah, makes sense.